docs/runbooks/SSPCommonTemplatesModificationReverted.md

```diff
@@ -0,0 +1,19 @@
+# SSPCommonTemplatesModificationReverted
+
+## Meaning
+
+This alert indicates that one or more common templates were changed by the ssp operator as part of its reconciliation procedure.
+
+## Impact
+
+If any changes were made to a common template manually or by a script they were reverted by the operator.
+
+## Diagnosis
```

Comment:
Is there a way to know which template was reverted? Are you putting them into metric labels? If so, we can state how to find the reverted template here.

Reply:
not currently but I am planning to add it, once it is in I will submit a PR to improve the runbook.